iquip/apps/logger.py

```diff
@@ -125,52 +137,75 @@ class LoggerApp(qiwis.BaseApp):
     Attributes:
         loggerFrame: A frame that shows the logs.
         confirmFrame: A frame that asks whether to clear logs.
-        handler: A handler for adding logs to the loggerFrame. 
+        levelsDict: A dictionary that matches int level to string level.
+        frameHandler: A handler for adding logs to the loggerFrame.
+        fileHandler: A handler for saving logs to file.
     """
 
-    def __init__(self, name: str, parent: Optional[QObject] = None):
-        """Extended."""
+    def __init__(self, name: str, path: str, parent: Optional[QObject] = None):
+        """Extended.
+        
+        Args:
+            name: Specifies name of the LoggerApp.
+            path: Desribes file directory of log record file.
+        """
         super().__init__(name, parent=parent)
         self.loggerFrame = LoggerFrame()
         self.confirmFrame = ConfirmClearingFrame()
         # connect signals to slots
         self.loggerFrame.clearButton.clicked.connect(self.checkToClear)
         self.confirmFrame.confirmed.connect(self.clearLog)
-        self.handler = LoggingHandler(self.addLog)
-        # TODO(aijuh): Change the log format when it is determined.
-        fs = "%(levelname)s [%(name)s] [%(filename)s:%(lineno)d] %(message)s"
-        formatter = logging.Formatter(fs)
-        self.handler.setFormatter(formatter)
+        # initialize handlers
+        self.levelsDict = {logging.DEBUG: "DEBUG", logging.INFO: "INFO", logging.WARNING: "WARNING",
+                           logging.ERROR: "ERROR", logging.CRITICAL: "CRITICAL"}
+        self.frameHandler = LoggingHandler(self.addLog)
+        logFileName = os.path.join(path, QDateTime.currentDateTime().toString("logyyMMdd-HHmmss"))
```

Comment:
How about adding "_" between "log" and the datetime?

Reply:
Also, please use the extension ".log".

Reply:
Also, you set the `when` parameter of the file handler to "midnight", so isn't it unnecessary the time part in file name.
Isn't "log_yyMMdd.log" enough?

Reply:
I am not sure about the naming convention about time.
As I turn and turn off qiwis while testing, it seems helpful to distinguish current file.
But in real application, I understand that qiwis is keep running and not turn off. In this case time part seems unnecessary.

Reply:
The file name is created when the program starts, i.e., the filename shows that the start time. Note that we can run the program several times in a single day.

Reply:
> Note that we can run the program several times in a single day.

@kangz12345 
Then, why do we set `when` parameter of `TimedRotatingFileHandler` to "midnight"?
If the log file is created everytime the program starts, I think the `TimedRotatingFileHandler` is unnecessary.

Reply:
Because we usually don't terminate the GUI program. Then a single log file would become too large, which make handling difficult.

Reply:
Okay! Now I see.